app/Providers/RouteServiceProvider.php

```diff
@@ -207,6 +207,20 @@ private function mapAdminRoutes()
 
             Route::resource('fares', 'FareController')->middleware('ability:admin,finances');
 
+            // downloads
+            Route::group([
+                'as'         => 'downloads.',
+                'prefix'     => 'downloads',
+                'middleware' => ['ability:admin, files'],
+            ], function () {
+                Route::get('/', 'DownloadController@index')->name('index');
+                Route::get('/create', 'DownloadController@create')->name('create');
+                Route::get('/edit/{id}', 'DownloadController@edit')->name('edit');
+                Route::post('/', 'FileController@store')->name('store');
```

Comment:
These are going to break the existing FileController rules

Reply:
Oh, so I copy thr `FileController `store to `Admin\DownloadController`?

Reply:
FileController is already being used, moving it here into this namespace will break it

Reply:
Umm so you suggest that I should create a DownloadService and use it to store downloads?

Reply:
No... I haven't entirely thought through this feature yet

Reply:
Oh, okay.
Let me assure you that for now it works fine.